main/upower/APKBUILD

```diff
@@ -1,17 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=upower
-pkgver=0.99.7
+pkgver=0.99.10
+_distver="$(echo ${pkgver} | tr '.' '_')"
 pkgrel=0
 pkgdesc="Power Management Services"
 url="https://upower.freedesktop.org"
 arch="all"
 license="GPL-2.0-or-later"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev
-	dbus-glib-dev libxslt gobject-introspection-dev docbook-xsl"
-source="https://upower.freedesktop.org/releases/upower-$pkgver.tar.xz"
-# XXX: tests are broken on alpine for some reason, but not on adelie
-options="!check"
+makedepends="linux-headers gtk+-dev libgudev-dev libusb-dev polkit-dev gtk-doc
+	dbus-glib-dev libxslt gobject-introspection-dev docbook-xsl autoconf automake
+	py3-dbus"
+source="https://gitlab.freedesktop.org/upower/upower/-/archive/UPOWER_${_distver}/upower-UPOWER_${_distver}.tar.bz2"
+builddir="$srcdir/$pkgname-UPOWER_${_distver}"
```

Comment:
```suggestion
builddir="${srcdir}/${pkgname}-${_pkgname}"
```

Reply:
Uh? We don't seem to use curly braces for these elsewhere

Reply:
Yes, some Alpine Linux folks are anti braces. My advice is to always use them, it's a tiny bit less to track in your head when figuring out which characters will give you an unset variable.

Reply:
I'm totally with you there, I rather place braces than debug weird errors, but maybe it'd make sense to implement some common guide line for this? I've been told in a review before to remove them, so it kind of sucks when I have to guess what "kind" of reviewer will look at my stuff :)

Reply:
I totally agree with you. So many of the reviewer requests seem to come out of nowhere. We should certainly have them written down, but changes in this area have been tremendously slow in my experience.

Reply:
I prefer to only use braces where it makes sense to use them not everywhere. But I respect that others may disagree so I'm not gonna stand in the way if a package maintainer insists on braces everywhere in APKBUILDs he/she maintains.